RallySimulator.Application/Core/Vehicles/Queries/GetVehicles/GetVehiclesQuery.cs

```diff
@@ -145,14 +145,14 @@ private static string ValidateOrderBy(string orderBy)
             {
                 string columnName = orderByPart.Trim(' ').Split(' ')[0].ToLower();
 
-                if (!ValidOrderByColumnsDictionary.ContainsKey(columnName))
+                if (!ValidOrderByColumnsDictionary.TryGetValue(columnName, out string value))
```

Comment:
How about instead of `value` we use something descriptive like `orderByColumn`?

Reply:
Done